aws-postgresql.yml

```diff
@@ -364,6 +372,15 @@ provision:
   - field_name: password
     type: string
     details: The password to authenticate to the database instance.
+  - field_name: use_managed_admin_password
+    type: boolean
+    details: Whether CSB should use AWS managed admin password
+  - field_name: master_secret_arn
```

Comment:
I'm wondering whether the "master secret" is the same thing as the "admin password". This will no doubt get clearer as I read on. If they are the same concept, then it would be nice for the names to help maintainers understand that they are the same concept.

Reply:
Coming back to this, I think they are the same concept. Could we call this something like `managed_admin_password_arn` or `managed_admin_credentials_arn`?